server/startup/migrations/v033.js

```diff
@@ -0,0 +1,78 @@
+RocketChat.Migrations.add({
+	version: 33,
+	up() {
+		const scriptAlert = '/**\n * This scrit is out-of-date, convert to the new format\n * (https://github.com/RocketChat/Rocket.Chat/wiki/WebHook-Scripting)\n**/\n\n';
```

Comment:
Fix the misspelling `scrit` to `script`. And change the url to: https://rocket.chat/docs/administrator-guides/integrations